manifest.json

```diff
@@ -420,11 +430,12 @@
         "targetDir": "midi-js-a8a842"
       },
       "ngJoyride": {
-        "version": "5b25d076405c9a787db602d2a8ff49919228f4fc",
-        "downloadFormat": "zip",
-        "url": "https://github.com/seanlip/ng-joyride/archive/5b25d076405c9a787db602d2a8ff49919228f4fc.zip",
+        "version": "b117e0",
+        "downloadFormat": "files",
+        "url": "https://raw.githubusercontent.com/oppia/ng-joyride/master/",
```

Comment:
Do not use master here, use specific version (like in https://github.com/oppia/oppia/blob/develop/manifest.json#L501).

Reply:
Done Thanks!